### PR TITLE
Add i18n to ErrorMessage.js LoggingIn.js CreationSucessPage.js

### LINKED ISSUE #271: Add i18n: CreationSuccessPage
We'd like to make the newdle frontend i18n-capable.
This issue is about adding i18n support to the `CreationSuccessPage` component (usually this is in a file with the same name).

Please test your changes, especially if you update a string that needs pluralization (we did not add pluralization so far because we didn't have i18n and doing it without an i18n library is not pretty).

Useful resources:

- The project's [README](https://github.com/indico/newdle#readme) explains how to run newdle
- **[Here's an example PR](https://github.com/indico/newdle/pull/267/files) where we added i18n to some components.** It contains [React usage](https://github.com/indico/newdle/pull/267/files#diff-0cc54461cea891fa78c0e85c7d6df434R172) (`<Trans>`), [string usage](https://github.com/indico/newdle/pull/267/files#diff-e5661acb3a7ba762bb8141855a9ccf73R191) (template literal tagged with `t`) and [pluralization](https://github.com/indico/newdle/pull/267/files#diff-0cc54461cea891fa78c0e85c7d6df434R153-R157).
- There's also the [Lingui documentation](https://js-lingui-git-next.lingui-js.now.sh/) but be aware that this is not fully up to date for the version we are using (Lingui 3)
- We do not use artificial IDs (like `newdle.created.message`); the source strings are always the English version
- When adding the lingui-related import (from `'@lingui/macro'`) make sure to put this import before any relative `./something` imports
- You do not need to update any .po files manually; you can run `make i18n` or simply `git commit` - we have a pre-commit hook which automatically updates the po file and stages it to be committed as well.

In case the component in this issue is rather small, feel free to take care of multiple components in a single PR. Don't forget to add `closes #xxx, closes #yyy` to your PR so all relevant issues are closed.

Thanks in advance for making a useful contribution during this [Hacktoberfest](https://hacktoberfest.digitalocean.com/)! 🎉

### LINKED ISSUE #272: Add i18n: ErrorMessage
We'd like to make the newdle frontend i18n-capable.
This issue is about adding i18n support to the `ErrorMessage` component (usually this is in a file with the same name).

Please test your changes, especially if you update a string that needs pluralization (we did not add pluralization so far because we didn't have i18n and doing it without an i18n library is not pretty).

Useful resources:

- The project's [README](https://github.com/indico/newdle#readme) explains how to run newdle
- **[Here's an example PR](https://github.com/indico/newdle/pull/267/files) where we added i18n to some components.** It contains [React usage](https://github.com/indico/newdle/pull/267/files#diff-0cc54461cea891fa78c0e85c7d6df434R172) (`<Trans>`), [string usage](https://github.com/indico/newdle/pull/267/files#diff-e5661acb3a7ba762bb8141855a9ccf73R191) (template literal tagged with `t`) and [pluralization](https://github.com/indico/newdle/pull/267/files#diff-0cc54461cea891fa78c0e85c7d6df434R153-R157).
- There's also the [Lingui documentation](https://js-lingui-git-next.lingui-js.now.sh/) but be aware that this is not fully up to date for the version we are using (Lingui 3)
- We do not use artificial IDs (like `newdle.created.message`); the source strings are always the English version
- When adding the lingui-related import (from `'@lingui/macro'`) make sure to put this import before any relative `./something` imports
- You do not need to update any .po files manually; you can run `make i18n` or simply `git commit` - we have a pre-commit hook which automatically updates the po file and stages it to be committed as well.

In case the component in this issue is rather small, feel free to take care of multiple components in a single PR. Don't forget to add `closes #xxx, closes #yyy` to your PR so all relevant issues are closed.

Thanks in advance for making a useful contribution during this [Hacktoberfest](https://hacktoberfest.digitalocean.com/)! 🎉

### LINKED ISSUE #276: Add i18n: LoggingIn
We'd like to make the newdle frontend i18n-capable.
This issue is about adding i18n support to the `LoggingIn` component (usually this is in a file with the same name).

Please test your changes, especially if you update a string that needs pluralization (we did not add pluralization so far because we didn't have i18n and doing it without an i18n library is not pretty).

Useful resources:

- The project's [README](https://github.com/indico/newdle#readme) explains how to run newdle
- **[Here's an example PR](https://github.com/indico/newdle/pull/267/files) where we added i18n to some components.** It contains [React usage](https://github.com/indico/newdle/pull/267/files#diff-0cc54461cea891fa78c0e85c7d6df434R172) (`<Trans>`), [string usage](https://github.com/indico/newdle/pull/267/files#diff-e5661acb3a7ba762bb8141855a9ccf73R191) (template literal tagged with `t`) and [pluralization](https://github.com/indico/newdle/pull/267/files#diff-0cc54461cea891fa78c0e85c7d6df434R153-R157).
- There's also the [Lingui documentation](https://js-lingui-git-next.lingui-js.now.sh/) but be aware that this is not fully up to date for the version we are using (Lingui 3)
- We do not use artificial IDs (like `newdle.created.message`); the source strings are always the English version
- When adding the lingui-related import (from `'@lingui/macro'`) make sure to put this import before any relative `./something` imports
- You do not need to update any .po files manually; you can run `make i18n` or simply `git commit` - we have a pre-commit hook which automatically updates the po file and stages it to be committed as well.

In case the component in this issue is rather small, feel free to take care of multiple components in a single PR. Don't forget to add `closes #xxx, closes #yyy` to your PR so all relevant issues are closed.

Thanks in advance for making a useful contribution during this [Hacktoberfest](https://hacktoberfest.digitalocean.com/)! 🎉

### PR DESCRIPTION
**Pull request by ruhci28**
Closes #272 , Closes #276 , Closes #271

### CODE REVIEW

File: newdle/client/src/components/ErrorMessage.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React, {useEffect} from 'react';
 import {useDispatch} from 'react-redux';
 import PropTypes from 'prop-types';
 import {useHistory} from 'react-router';
+import {Trans, Plural, t} from '@lingui/macro';
```
Comment (ThiefMaster) [inline newdle/client/src/components/ErrorMessage.js:5]: Please remove the unused imports. You are only using `t` in here.

```suggestion
import {t} from '@lingui/macro';
```

**Review by ThiefMaster — COMMENTED** [review]
Thanks for the PR! Just one small thing to fix..

File: newdle/client/src/components/ErrorMessage.js (hunk 1 above)
Comment (ruhci28) [reply newdle/client/src/components/ErrorMessage.js:5]: ok

File: newdle/client/src/components/login/LoggingIn.js [hunk 2]
```diff
@@ -1,10 +1,11 @@
 import React from 'react';
+import {Trans} from '@lingui/macro';
 import {Dimmer, Loader} from 'semantic-ui-react';
 
 export default function LoggingIn() {
   return (
     <Dimmer active page>
-      <Loader size="massive">Logging in...</Loader>
+      <Loader size="massive"><Trans>Logging in...</Trans></Loader>
```
Comment (ThiefMaster) [inline newdle/client/src/components/login/LoggingIn.js:8]: Prettier wants to reformat this:

```suggestion
      <Loader size="massive">
        <Trans>Logging in...</Trans>
      </Loader>
```

Comment (ruhci28) [reply newdle/client/src/components/login/LoggingIn.js:8]: ok sure

**Comment by ThiefMaster** [comment]
Please checkout the repo locally, run `make` and `make i18n`.

Currently your PR does not contain the updates to the `.po` file and still violates the formatting rules of *prettier*.

**Comment by ThiefMaster** [comment]
there are still linter errors. please run `make format i18n` to fix those.

**Comment by ruhci28** [comment]
@ThiefMaster can you review the PR now

**Review by ThiefMaster — APPROVED** [review]
Thanks!

**Pull request merged**